dashboard/app/views/devise/registrations/edit.html.haml

```diff
@@ -287,7 +219,27 @@
       isCleverStudent: current_user.clever_student?,
       dependentStudentsCount: Queries::User.dependent_students_count(current_user.id),
       personalAccountLinkingEnabled: @personal_account_linking_enabled,
-      lmsName: Queries::Lti.get_lms_name_from_user(current_user)
+      lmsName: Queries::Lti.get_lms_name_from_user(current_user),
+      authorizedTeacher: current_user.permission?(UserPermission::AUTHORIZED_TEACHER),
```

Comment:
There's a `verified_teacher?` method in the User model that checks for this permission. I think we generally use the term "verified" instead of "authorized," even though the permission is called AUTHORIZED_TEACHER, to avoid confusion with actual authorization logic. Minor nitpick, but that would probably be the better name for the variable.

Reply:
nice, I'll use that method and change the variable name